hw/top_earlgrey/data/ip/chip_otp_ctrl_testplan.hjson

```diff
@@ -0,0 +1,208 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: chip_otp_ctrl
+  testpoints: [
+    // OTP (pre-verified IP) integration tests:
+    {
+      name: chip_otp_ctrl_init
+      desc: '''Verify the OTP ctrl initialization on chip power up.
+
+            Verify that the chip powers up correctly on POR.
+            - The pwrmgr initiates a handshake with OTP ctrl and later, with LC ctrl in subsequent
+              FSM states. Ensure that the whole power up sequence does not hang.
+            - Verify with connectivity assertion checks, the handshake signals are connected.
+            - Ensure that no interrupts or alerts are triggered.
+            '''
+      features: ["OTP_CTRL.INIT"]
+      stage: V2
+      si_stage: SV1
```

Comment:
You can probably add a manufacturing test bazel target here.

Reply:
will do once manufacturing tests are available. For now I can only only a sets and actively updated.